ui/src/app/components/card.tsx

```diff
@@ -5,14 +5,16 @@ const cardBorderColor = colorWithWhiteness(colors.dark, 0.6);
 
 export const baseStyles = reactStyles({
   card: {
-    padding: '1rem',
+    margin: '0 1rem 1rem 0',
     borderRadius: '0.2rem',
     boxShadow: `0 0.125rem 0.125rem 0 ${cardBorderColor}`,
     backgroundColor: colors.white,
     border: `1px solid ${cardBorderColor}`,
     display: 'flex',
-    flexDirection: 'column',
-    margin: '0 1rem 1rem 0'
+    flexDirection: 'column'
+  },
+  cardPadding: {
```

Comment:
Since `workspaceCard` is the only exception to the rule on this, I'd probably rather have that override the padding value, and leave the default padding as is.